cypress/support/commands.ts

```diff
@@ -7,79 +7,98 @@
  * be emitted first before any other mouse event. Hence, we scroll the element into view, wait for 16ms and
  * eventually trigger the mouse event.
  */
-Cypress.Commands.add('dispatch', { prevSubject: 'element' }, (subject, event, ...args): any => {
-  return cy
-    .wrap(subject)
-    .scrollIntoView()
-    .wait(16)
-    .trigger(event, ...args)
-    .wait(16);
-});
-
-Cypress.Commands.add('shouldSelect', (ids: Array<number>, ...args): any => {
-  const expected = ids.map(id => `Document ${id}`);
-
-  const checkSelectedItems = (items: Cypress.ObjectLike): { success: boolean; actual: Array<string> } => {
-    const actual: Array<string> = Array.prototype.map.call(items, (item: HTMLElement) => item.textContent);
-
-    const success = expected.every(expectedTextContent =>
-      actual.some(actualTextContent => actualTextContent === expectedTextContent)
-    );
-
-    return {
-      success,
-      actual
-    };
-  };
-
-  return cy
-    .getSelectedItems()
-    .should('have.length', ids.length)
-    .then(checkSelectedItems)
-    .then(result => {
-      if (!result.success) {
-        Cypress.log({
-          message: `Selected items do not match`,
-          consoleProps: () => [`Expected: ${expected}`, `Actual: ${result.actual}`]
-        });
-      }
-
-      return result.success;
-    })
-    .should('eq', true);
-});
-
-Cypress.Commands.add('getSelectItem', (index: number, ...args): any => {
-  let options: any = {};
-
-  if (Cypress._.isObject(args[0])) {
-    options = args[0];
+Cypress.Commands.add(
```

Comment:
Is this formatting introduced by a newer version of Prettier? Looks super ugly if you ask me.